mythril/ethereum/evmcontract.py

```diff
@@ -23,6 +27,15 @@ def __init__(
         self.creation_disassembly = Disassembly(
             creation_code, enable_online_lookup=enable_online_lookup
         )
+        try:
```

Comment:
I think this fits better as an @property function. Separates this logic from the constructor and allows us to create a well documented acessor